macros.zig_extra

```diff
@@ -21,8 +21,8 @@
 # -r: "Release." Choose the build/release mode Zig will use. Note that "safe" is the default set by the Zig macros so it is never necessary to specify this option.
 %zig_build_target(cr:) %{lua:
    if opt.c then
-    if rpm.expand("%?1"):find("%w") then
-     rpm.define("_zig_cpu " .. rpm.expand("%1"))
+    if rpm.expand("%{?1}"):find("%w") then
+     rpm.define("_zig_cpu " .. rpm.expand("%{?1:%{1}}"))
```

Comment:
well it's guaranteed %1 is defined, so you only need to use %1 here instead of %{?1:%1}